vault/changelog.release.2022-03-15.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 4i7t856tu1wygd2o540lc6e
+title: '0.86'
+desc: ''
+updated: 1647297632209
+created: 1647295787428
+---
+
+Dendron 0.86 has sprouted  🌱
+
+The Airtable Export Pod now supports null entries for dates. Before this, an empty/null value for date would prompt issues when attempting to export notes to Airtable. Learn more here: ([[docs|dendron://dendron.dendron-site/dendron.topic.pod-v2.ref.builtin.airtable.export#date]]) 
+
+This release includes a variety of other fixes and improvements.
+
+## Highlights
+- enhance(pods): support null entries for dates in Airtable export pod v2 
+
+## Everything Else
+- enhance(workspace): Dendron still loads if there's a missing local vault
+- fix(pods): Google Docs Export pod displays Bad Request error on export
```

Comment:
we snuck in a few more changes for this week. see below. the perf improvement is significant so could be worth highlighting

- enhance(basics): improve performance around reference rendering including hover #2543 @jonathan
- enhance(workspace): Dendron still loads if there's a missing local vault (#2526) @kaan
- fix(basics): ensure note title is always a string to avoid errors (#2551) @kaan

Reply:
all sounds good. how should I go about updating the PR?

Reply:
@jpknwls
- Add the `enhance(*` lines under the `### Enhancements` portion of the changelog, and `fix*` to the `### Fix` section
- Add the `improve performance` enhancement to the Highlights line items in the release notes, and the other two to the Everything Else
- Add a sentence to the highlights content showcasing the updates in this release to include the improvement of performance